internal/stack/_static/elasticsearch.yml.tmpl

```diff
@@ -13,6 +13,12 @@ xpack.security.http.ssl.certificate: "certs/cert.pem"
 
 ingest.geoip.downloader.enabled: false
 
+{{- $version := fact "elasticsearch_version" -}}
```

Comment:
Nit: elasticsearch version is also fetched on line 22. I was also wondering about the diff with `-` vs without.

Reply:
Ah, thanks, I will remove the line below. The `-` is to avoid adding newlines when rendering the template.